master/releases/index.md

```diff
@@ -12,7 +12,11 @@ Use the version selector at the top-right of this page to view a different relea
 [Release archive](https://github.com/projectcalico/calico/releases/download/{{ release.title }}/release-{{ release.title }}.tgz) with Kubernetes manifests, Docker images and binaries.
 {% endunless %}
 
+{% if release.note %}
```

Comment:
I'd recommend turning this page into more of an index, rather than containing the info embedded. The corresponding GitHub release can have the same content.

e.g. a few different links:

```
- See the release notes
- Download release artifacts
- Upgrade procedure
```

Reply:
As part of this, I'd really like to see this go into a top-level directory called `release_notes/` rather than buried in _includes.